### ISSUE TITLE
IDISP003 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#53]
```cs
namespace RoslynSandbox
{
    using System.IO;

    public class Foo
    {
        public static bool TryGetStream(out Stream stream)
        {
            return TryGetStreamCore(out stream);
        }

        private static bool TryGetStreamCore(out Stream stream)
        {
            stream = File.OpenRead(string.Empty);
            return true;
        }
    }
}
```

**Comment by JohanLarsson** [comment]
Fixed.
